src/data/matrix/kronecker.lean

```diff
@@ -290,6 +300,14 @@ lemma diagonal_kronecker_tmul_diagonal
   (diagonal a) ⊗ₖₜ[R] (diagonal b) = diagonal (λ mn, a mn.1 ⊗ₜ b mn.2) :=
 kronecker_map_diagonal_diagonal _ (zero_tmul _) (tmul_zero _) _ _
 
+
+variables [add_comm_monoid γ] [module R γ]
```

Comment:
Can you put these up on line 248?

Reply:
Sure, done. I thought it would make the code easier to read if they would intervene only where needed.